python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -11,8 +11,9 @@
 import time
 import typing as t
 from dataclasses import dataclass, field
-
+from typing import Self
```

Comment:
blocking: Sadly, as we discovered, we won't be able to support Self until 2026!

Reply:
accidentally committed, reverted.